File: backend-fastapi/app/api/health.py
from fastapi import APIRouter
from datetime import datetime
import time
import os

router = APIRouter()

start_time = time.time()

@router.get("/health")
async def health_check():
    return {
        "success": True,
        "message": "Server is healthy",
        "timestamp": datetime.utcnow().isoformat(),
        "uptime": time.time() - start_time,
        "environment": os.getenv("NODE_ENV", "development")
    }

File: backend/src/routes/health.ts
import { Router, Request, Response } from 'express';

const router = Router();

router.get('/', (_req: Request, res: Response) => {
  res.status(200).json({
    success: true,
    message: 'Server is healthy',
    timestamp: new Date().toISOString(),
    uptime: process.uptime(),
    environment: process.env.NODE_ENV || 'development',
  });
});

export default router;
